services/newrelic/newrelic.ini

```diff
@@ -191,14 +186,14 @@ thread_profiler.enabled = true
 #
 
 [newrelic:development]
-monitor_mode = false
+monitor_mode = true
 
 [newrelic:test]
-monitor_mode = false
+monitor_mode = true
 
 [newrelic:staging]
 app_name = Python Application (Staging)
```

Comment:
This should be changed to "Perma (Staging)". Alternatively you can use the environment variable NEW_RELIC_APP_NAME